packages/scripts/src/SimpleTextGenerator.ts

```diff
@@ -0,0 +1,49 @@
+import { OpenAI } from "mongodb-rag-core/openai";
```

Comment:
i recommend using the AI SDK's function `generateText()` for this. wont need to make own constructor. can import from `mongodb-rag-core/aiSdk`. there's lots of examples throughout the project for usage